### ISSUE TITLE
Port the facebook automation pipeline to use the new codebase

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/covidcast-indicators#249]
Remaining tasks:

- [x] Add wrapper to complete pipeline (qualtrics download, weights download, CIDs upload, micro upload, optional validation)
- [x] Install wrapper to run in parallel in cron
- [x] verify new pipeline works as designed
- [x] Generate new issue of COVIDcast CSVs for the  complete survey record going back to 6 April
- [x] Generate new issue of micro data for the complete survey record going back to 6 April?
- [x] Install/submit issues and flip the switch
